package.json

```
@@ -23,5 +23,19 @@
   "homepage": "https://github.com/digitalbazaar/web-credential-handler",
   "bedrock": {
     "browserDependencies": "all"
-  }
+  },
+  "scripts": {
+    "prepublish": "npm run build",
+    "build": "webpack -p"
```

Move mode to config file.
```suggestion
    "build": "webpack"
```